napari/utils/theme.py

```diff
@@ -180,4 +180,4 @@ def available_themes():
     list of str
         Names of available themes.
     """
-    return list(_themes)
+    return tuple(_themes)
```

Comment:
This should probably be reverted, or the docstring updated, but that can be done in a followup PR.